mmir-plugin-asr-bingWS-web/example/config/configuration.json

```diff
@@ -0,0 +1,47 @@
+{
+    "language": "en",
+    "usePrecompiledViews": "true",
+    "usePrecompiledGrammarsOnly": "true",
+    "grammarCompiler": "jscc",
+    
+    
+    "maryTextToSpeech": {
+    	"serverBasePath": "http://mary.dfki.de:59125/"
+    },
+    
+    "mediaManager": {
+    	"plugins": {
+    		"browser": ["html5AudioOutput",
+    		            {"mod": "webAudioInput", "config": "webasrNuanceImpl"},

```

Comment:
the example should either be adjusted to use _this_ plugin (or its folder should be removed)